plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityTrapTime.java

```diff
@@ -0,0 +1,83 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.entity.SkeletonHorse;
+
+public class EntityTrapTime implements Property {
+
+    public static boolean describes(ObjectTag entity) {
+        return entity instanceof EntityTag ent &&
+                (ent.getBukkitEntity() instanceof SkeletonHorse);
+    }
+
+    public static EntityTrapTime getFrom(ObjectTag entity) {
+        if (!describes(entity)) {
+            return null;
+        }
+        else {
+            return new EntityTrapTime((EntityTag) entity);
+        }
+    }
+
+    private EntityTrapTime(EntityTag ent) {
+        entity = ent;
+    }
+
+    EntityTag entity;
+
+    @Override
+    public String getPropertyString() {
+        return String.valueOf(((SkeletonHorse) entity.getBukkitEntity()).getTrapTime());

```

Comment:
This will hold the wrong value. Check what `.describe` outputs and make sure it outputs valid input